docs/Splashkit/Applications/Arcade Machines/Arcade Machine Setup/Setup Arcade Machine.md

```diff
@@ -295,7 +302,7 @@ and one to wpa_supplicant:
 
 4. Select Console Autologin
 
-   ![Rasberry Pi Boot Option Screen](/docs/Splashkit/Applications/Arcade%20Machines/Arcade%20Machine%20Setup/Images/ConsoleAutoLogin.png)
+   ![Rasberry Pi system Option Screen](/docs/Splashkit/Applications/Arcade%20Machines/Arcade%20Machine%20Setup/Images/ConsoleAutologin.png)
```

Comment:
I'd prefer system to capitalised and therefore consistent with the rest of the phrase/sentence.

Reply:
Good call. I ended up changing this back to "Rasberry Pi Boot Option Screen" as this is what it was embedded as in the previous iteration of the document. When I was playing around with the markdown file to get it working I copied out the link from the _next_ item, so the title ended up getting duplicated. 

![image](https://github.com/user-attachments/assets/f9600e06-c5da-4457-93e9-3aee7ed1a8fb)